src/features/channels/components/channel-list/about/about.styles.ts

```diff
@@ -0,0 +1,50 @@
+import { createStyles, makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    iconContainer: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'end',
```

Comment:
All the classes here except for the 'term' class are unused, please remove them

Reply:
That's right